src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -6,19 +6,89 @@
  * За бажанням можна реалізувати інші методи інтрефейсу Map.</p>
  */
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75F;
+    private int size;
+    private Node<K, V>[] table;
+
+    public MyHashMap() {
+        size = 0;
+        table = (Node<K, V>[]) new Node[DEFAULT_CAPACITY];
+    }
 
     @Override
     public void put(K key, V value) {
-
+        resize();
+        if (addToTable(key, value, table)) {
```

Comment:
let's make addToTable void and increment size inside it

Reply:
done